netbox/extras/constants.py

```diff
@@ -135,3 +137,12 @@
         }
     },
 ]
+
+LOG_LEVEL_RANK = {
+    LogLevelChoices.LOG_DEFAULT: 0,
+    LogLevelChoices.LOG_DEBUG: 1,
```

Comment:
The existing LogLevelChoices has default defined so it needs to be mapped, we could filter and remove it from the dropdown but I included it because it is already existing.  If we removed it from LogLevelChoices then we would need a migration script to convert old script runs to remove it.